css/style.css

```diff
@@ -0,0 +1,71 @@
+body, p, div, header, footer {
+    margin: 0px;
+    padding: 0px;
+}
+div, header, p {
+    /*
+    border: 1px solid #2b2b2b;
+    */
+}
+
+.wrapper {
+    width: 1200px;
+    margin: auto;
+    font-size: 0;
+}
+.block {
+    height: 150px;
+    display: inline-block;
+    line-height: 150px;
+    vertical-align: middle;
+    font-size: 14px;
+}
+header .block{
+    width: 400px;
+}
+header, footer, .content-wrapper {
+    height: 150px;
+    min-width: 1200px;
```

Comment:
you now header can be used inside main and so on
please think where `min-width: 1200px` could be moved